File: Algorithms/01-Warmup/08-birthday-cake-candles/08.rb
#!/usr/bin/env ruby

def birthday_cake_candles(ar)
    mymax = ar.max
    count = 0
    ar.each do |x|
        count += 1 if x == mymax
    end
    count
end

n = gets.strip.to_i
ar = gets.strip
ar = ar.split(" ").map(&:to_i)
result = birthday_cake_candles(ar)
puts result

File: Algorithms/01-Warmup/08-birthday-cake-candles/08.py
#!/usr/bin/env python


def birthdayCakeCandles(ar):
    count = 0
    mymax = max(ar)
    for x in ar:
        if x == mymax:
            count += 1
    return count


n = int(input().strip())
ar = list(map(int, input().strip().split(' ')))
result = birthdayCakeCandles(ar)
print(result)
